tests/test-recipes/split-packages/test-file-hash/build.sh

```diff
@@ -0,0 +1,8 @@
+echo "int main() {}" > main.c
+mkdir -p $PREFIX/bin
+$CC main.c -o $PREFIX/bin/_file_hash
```

Comment:
@conda/builds-tools any idea why `$CC` is not set in the test?

Reply:
`testing_config` has `activate=False` by default, we have to enable it manually. I moved the test case to a separate test function to special case it.